sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -0,0 +1,61 @@
+## API Report File for "@azure/core-lro"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public (undocumented)
```

Comment:
do we want to handle all these nits in the initial PR?

Reply:
oh this file is auto-generated. I can omit this file from the PR for sure!

Reply:
I think we usually check this file in and use it to validate our reference docs are fleshed out, right?